core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -191,12 +191,13 @@ object TopicCommand extends Logging {
     val topics = getTopics(zkUtils, opts)
     val reportUnderReplicatedPartitions = if (opts.options.has(opts.reportUnderReplicatedPartitionsOpt)) true else false
     val reportUnavailablePartitions = if (opts.options.has(opts.reportUnavailablePartitionsOpt)) true else false
+    val reportUnbalancedPartitions = if (opts.options.has(opts.reportUnbalancedPartitionsOpt)) true else false
     val reportOverriddenConfigs = if (opts.options.has(opts.topicsWithOverridesOpt)) true else false
```

Comment:
I haven't reviewed the PR, but can I just say that all of these `if/else` are completely unnecessary? It's enough to say `opts.options.has(opts.reportUnderReplicatedPartitionsOpt)` for example.